### PR TITLE
Print version (QC-234)

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

File: Framework/src/TaskRunner.cxx [hunk 1]
```diff
@@ -57,6 +57,9 @@ TaskRunner::TaskRunner(const std::string& taskName, const std::string& configura
     mCycleNumber(0),
     mTotalNumberObjectsPublished(0)
 {
+  // Log the version
```
Comment (knopers8) [inline Framework/src/TaskRunner.cxx]: Can we put it at the beginnings of `InfrastructureGenerator::generateLocalInfrastructure()` and `InfrastructureGenerator::generateRemoteInfrastructure()` instead? These are usually the first QC functions invoked in the code. So if something breaks before we reach the ctor of `TaskRunner`, then we would still see the version number.

Comment (Barthelemy) [reply Framework/src/TaskRunner.cxx]: excellent idea, I am preparing the change

File: Framework/src/InfrastructureGenerator.cxx [hunk 2]
```diff
@@ -128,7 +128,8 @@ void InfrastructureGenerator::customizeInfrastructure(std::vector<framework::Com
 void InfrastructureGenerator::printVersion()
 {
   // Log the version
-  QcInfoLogger::GetInstance() << "QC version " << o2::quality_control::core::Version::getString() << infologger::endm;;
+  QcInfoLogger::GetInstance() << "QC version " << o2::quality_control::core::Version::getString() << infologger::endm;
+  ;
```
Comment (knopers8) [inline Framework/src/InfrastructureGenerator.cxx]: why semicolon?

Comment (Barthelemy) [reply Framework/src/InfrastructureGenerator.cxx]: thanks for noticing it. It is just a typo.

**Comment by Barthelemy** [comment]
@ktf  here is a new instance of missing logs.

**Comment by ktf** [comment]
Can you force push to retrigger a build? AFAICT the builder is working correctly now...

**Comment by Barthelemy** [comment]
sure

**Comment by Barthelemy** [comment]
the failing test is unrelated

**Pull request merged**